### ISSUE TITLE
Common Mistake!  CRITICAL

### ISSUE THREAD

**Issue by JaimeGomezSPE** [krishnandusarkar/SPE-v2#81]
I scanned a code.....as I was reading the question, the APP crashed and closed.  I had to open it again (no sign in required)....but when I went to scan the code again to retrieve the question, I get the message that "This booth has already been scanned".  

I think we need to place a safegard that the booth is counted ONLY when the question is submitted and not when the QR code is read.....so we avoid that if the APP crashes in the middle of the question, the team has to skip the booth

**Comment by krishnandusarkar** [comment]
@JaimeGomezSPE Yes the app has been developed to record the booth visit even if user doesn't answers the question.

If you want it that way, how are you planning to handle user force closing the app?

Let's discuss and fix it in v3.
